frontend/src/components/donation_page/index.css

```diff
@@ -8,6 +8,12 @@
   line-height: 1.6em;
 }
 
+#donate-now {
+  padding-top: 20px;
+  text-align: center;
+  padding-bottom: 10px;
+}
+
```

Comment:
Lets remove these styles [here](https://github.com/the-difference-engine/ymim/blob/master/frontend/src/components/donation_page/index.css#L16-L21) since they are no longer being used

Reply:
I removed them, thanks!